src/scf_module.F90

```diff
@@ -479,6 +479,8 @@ subroutine scf(env, mol, wfn, basis, pcem, xtbData, solvation, &
       write(env%unit,scifmt) "-> SCF convergence ",scfconv, "Eh  "
       write(env%unit,scifmt) "-> wf. convergence ",qconv,   "e   "
       write(env%unit,dblfmt) "Broyden damping    ",set%broydamp,"    "
+      write(env%unit,intfmt) "net charge         ",set%ichrg
```

Comment:
Is this or mol%chrg actually used?

Reply:
yeah, in principle `mol%chrg` is set based on the  `set%ichrg` value, but it might be more logical to retrieve this information from `mol` directly.

Reply:
Thank you for the suggestion; I've updated the PR accordingly.

Reply:
how about the uhf information?